pkg/operator/ceph/object/admin.go

```diff
@@ -410,6 +408,22 @@ func periodWillChange(current, staged string) (bool, error) {
 	return (diff != ""), nil
 }
 
+// convert a cmp.Path to a path in json format like might be used with jq to query the item
+// e.g., {"a": {"b": "c"}} returns .a.b
+func toJsonPath(path cmp.Path) string {
+	out := ""
+	for _, step := range path {
+		mi, ok := step.(cmp.MapIndex)
+		if !ok {
+			// because we use a dumb map[string]interface{}, we only need to process map indexes,
+			// but the path has other node types because of how the json gets processed
+			continue
```

Comment:
So the step failed, but we continue to the next step in case it succeeds? Will skipping steps still result in a consistent format?

Reply:
It's not a failed step. It's just a node type that we don't care about.